sunpy/net/scraper.py

```diff
@@ -405,7 +344,7 @@ def _smallerPattern(self, directoryPattern):
                 return relativedelta(seconds=1)
             elif "%M" in directoryPattern:
                 return relativedelta(minutes=1)
-            elif any(hour in directoryPattern for hour in ["%H", "%I"]):
```

Comment:
%I refers to what pattern?

Reply:
12-hour format hours as zero-padded decimal numbers as in 01, 02, ..., 12